js/src/components/paid-ads/ads-campaign.js

```diff
@@ -25,8 +25,8 @@ import FaqsSection from './faqs-section';
 /**
  * Renders the container of the form content for campaign management.
  *
- * Please note that this component relies on an AdaptiveForm's context, so it expects
- * a context provider component (`AdaptiveForm`) to existing in its parents.
+ * Please note that this component relies on an CampaignAssetsForm's context and custom adapter,
+ * so it expects a `CampaignAssetsForm` to existing in its parents.
```

Comment:
❓  I can see `<AdsCampaign>` being rendered in `<AdsStepper>` which is rendered by `<SetupAdsForm>` using just `<AdaptiveForm>`. So seems like it's not true that  `<CampaignAssetsForm>`  is a required wrapping component for  `<AdsCampaign>`.

Reply:
Thanks for catching this. The form component in `SetupAdsForm` indeed needs to be changed to `CampaignAssetsForm` as well, otherwise the "Continue" button in the ads setup flow will always be disabled. Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1863/commits/716526c20f7b332baf1aafadd249c87d2ee8500f.